module.modulemap

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+ module S2N {
+     explicit module Api {
```

Comment:
Might be worth putting a comment on this file describing what uses it / why it exists.

Also, should adding this file be part of an automated refactor of 153 files? I'd at least put it in a separate commit so people can look at it without looking through the whole diff :)

Reply:
They’re both the necessary parts of being able to build S2n with clang modules so I grouped them together.